app/src/main/java/org/sopt/soptseminar_week1/view/FollowingListAdapter.kt

```diff
@@ -25,7 +35,7 @@ class FollowingListAdapter : RecyclerView.Adapter<FollowingListAdapter.Following
     override fun getItemCount(): Int = userList.size
 
     override fun onBindViewHolder(holder: FollowingUserViewHolder, position: Int) {
-        holder.onBind(userList[position])
+        holder.onBind(userList[position], holder.itemView.getContext())
```

Comment:
```suggestion
        holder.onBind(userList[position], holder.itemView.context)
```

- itemView의 context는 kotlin에서 property로 정의되어 있어서 context로 접근가능함
- 우선 시도는 좋아 근데 굳이 itemView의 context를 받아올 필요가 있을까?